packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -181,6 +181,15 @@ export interface TldrawEditorBaseProps {
 	 * Options for syncing the editor's camera state with the URL.
 	 */
 	deepLinks?: true | TLDeepLinkOptions
+
+	/**
+	 * Predicate for whether or not a shape should be hidden.
+	 *
+	 * Hidden shapes will not render in the editor, and they will not be eligible for hit test via
+	 * {@link Editor#getShapeAtPoint} and {@link Editor#getShapesAtPoint}. But otherwise they will
+	 * remain in the store and participate in all other operations.
+	 */
+	isShapeHidden?: TLEditorOptions['isShapeHidden']
```

Comment:
we'll get better docs here if we inline the method type instead of referencing it like this